Updates in TPM 3.1 (#8821)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineVoiceUser.md

  - ResultSize
- LicenceState 

*Deprecated Input Parameters*

> - Below input parameters have been deprecated due to low/zero usage:

Reviewer: ```suggestion
- The following input parameters have been deprecated due to low or zero usage:
```

```

### -Force

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -GetFromAAD

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -GetPendingUsers

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -PSTNConnectivity

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -SearchQuery

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### -Tenant

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```

> *This parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.

Reviewer: ```suggestion
*This parameter has been deprecated from Teams PowerShell Modules 3.0 and above due to limited usage*.
```

```

### CommonParameters

> *These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*

Reviewer: ```suggestion
*These parameter has been deprecated from Teams Powershell Modules 3.0 and above due to limited usage*.
```
